tools/generator/templates/champions/cxx/interface_cxx.cc.jinja2

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+// This file is generated. It contains the code to call the API functions from
+// the C++ language. Do not edit this file.
```

Comment:
You could use:

```
// {{ stechec2_generated }}
// This file contains the code to call the API functions from the C++ language.
```

Reply:
And what would stechec2_generated contain ? Just “This file is generated. Do not edit“ ?

Reply:
Yes, it's common to have strings like this to annotate generated files, i.e `ansible_generated` for ansible.

Reply:
I still have to add this then.